src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -46,6 +41,27 @@ public function __construct(
         $this->strategy = $strategy ?? new AffirmativeStrategy();
     }
 
+    public function getDecision(TokenInterface $token, array $attributes, mixed $object = null, bool $allowMultipleAttributes = false): AccessDecision
+    {
+        // Special case for AccessListener, do not remove the right side of the condition before 6.0
+        if (\count($attributes) > 1 && !$allowMultipleAttributes) {
+            throw new InvalidArgumentException(\sprintf('Passing more than one Security attribute to "%s()" is not supported.', __METHOD__));
+        }
+
+        if (method_exists($this->strategy, 'getDecision')) {
```

Comment:
similar question as above: is this if/else required? what if we keep only the else path?